package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "gcds-components",
+  "name": "gcds-components-repo",
   "version": "0.1.9",
   "description": "gcds components proof-of-concept",
```

Comment:
Aaaaand while you are in here: should we remove the `proof-of-concept` part from the description?

Reply:
Looking at that file, there is a lot I can remove now since the stencil project is in the `packages/web` directory.